### PR TITLE
Fix the bug that can't query JVM instance metrics

### PR DESCRIPTION
**Pull request by fgksgf**
Because the demo env have changed, I can't test CLI commands now.

Resolve apache/skywalking#5973

### CODE REVIEW

**Review by wu-sheng — COMMENTED** [review]
FYI, other reviewers, I am arguing the solution here, https://github.com/apache/skywalking/issues/5973#issuecomment-740510997

File: commands/interceptor/scope.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func ParseScope(name string) schema.Scope {
 		ret = schema.ScopeServiceRelation
 	} else if strings.HasPrefix(name, "service_instance_relation") {
 		ret = schema.ScopeServiceInstanceRelation
-	} else if strings.HasPrefix(name, "service_instance") {
+	} else if strings.HasPrefix(name, "service_instance") || strings.HasPrefix(name, "instance_") {
```
Comment (wu-sheng) [inline commands/interceptor/scope.go:34]: I think you should remove this prefix mechanism?

Comment (fgksgf) [reply commands/interceptor/scope.go:34]: Ok, I thought your "auto-scope logic" referred this..

Comment (wu-sheng) [reply commands/interceptor/scope.go:34]: Let me be more clear. `auto-scope logic` means the scope is based on the input parameter.

File: commands/interceptor/entity.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		ret = schema.ScopeServiceInstanceRelation
+	} else if ctx.String("destService") != "" {
+		ret = schema.ScopeServiceRelation
+	} else if ctx.String("endpointName") != "" {
+		ret = schema.ScopeEndpoint
+	} else if ctx.String("instanceName") != "" {
+		ret = schema.ScopeServiceInstance
+	} else if ctx.String("serviceName") != "" {
+		ret = schema.ScopeService
+	}
```
Comment (wu-sheng) [inline commands/interceptor/entity.go:66]: This rule makes sense to me. @kezhenxu94 Please review others.

**Review by kezhenxu94 — DISMISSED** [review]
LGTM

**Review by wu-sheng — APPROVED** [review]
LGTM. Please confirm whether remove `don't merge`.

**Comment by fgksgf** [comment]
> LGTM. Please confirm whether remove `don't merge`.

Tested locally, it works well.

**Pull request merged**